### PR TITLE
Fixed #1084 : History should be fully translatable

### LINKED ISSUE #1084: History should be fully translatable
image
Note that"'History" activity title should be translatable as well as the timestamp.

### PR DESCRIPTION
**Pull request by PrajwalM2212**
## Description

Made the history as well as sign in translatable

## Related issues and discussion
#1084

### CODE REVIEW

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -424,4 +424,10 @@
     <string name="preference_header_about" translatable="false">@string/action_about</string>
     <string name="preference_header_contributing">Contributing</string>
     <string name="not_support_books">Books are not currently supported by Open Food Facts</string>
+    <string name="seconds_ago">seconds ago</string>
+    <string name="minutes_ago">minutes ago</string>
+    <string name="hours_ago">hours ago</string>
+    <string name="days_ago">days ago</string>
+    <string name="scan_first_string">Your viewed product history will be listed here.</string>
+    <string name="scan_first_string_continued">This history is for your eyes only and is stored locally.</string>
```
Comment (Karljoones) [inline app/src/main/res/values/strings.xml]: This should be combined into one string, just stick a "\n" where there should be a line break.

**Comment by PrajwalM2212** [comment]
@Karljoones  I will make the change.

**Comment by PrajwalM2212** [comment]
@Karljoones Changes made . Thanks for the help.

File: app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -69,7 +69,6 @@
         </activity>
         <activity
             android:name=".views.LoginActivity"
-            android:label="Sign In"
```
Comment (teolemon) [inline app/src/main/AndroidManifest.xml]: It doesn't reappear in the string file ?

File: app/src/main/AndroidManifest.xml [hunk 3]
```diff
@@ -89,7 +88,6 @@
             android:screenOrientation="portrait" />
         <activity
             android:name=".views.HistoryScanActivity"
-            android:label="History"
```
Comment (teolemon) [inline app/src/main/AndroidManifest.xml]: It doesn't reappear in the string file ?

Comment (PrajwalM2212) [reply app/src/main/AndroidManifest.xml]: @teolemon The string is equivalent to "R.string.scan_history_drawer" which is translatable . Hence label has been removed and title has been set using "R.string.scan_history_drawer"  in HistoryScanActivity .

File: app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (PrajwalM2212) [reply app/src/main/AndroidManifest.xml]: @teolemon The string is equivalent to "R.string.txtSignIn" which is translatable . Hence label has been removed and title has been set using "R.string.txtSignIn" in LoginActivity.

**Comment by PrajwalM2212** [comment]
@teolemon @Karljoones  This is fixed

File: app/src/main/AndroidManifest.xml (hunk 2 above)
Comment (PrajwalM2212) [reply app/src/main/AndroidManifest.xml]: @teolemon 
cat2
cat

The title of category activity is set in the activity itself. There is no label for category activity in manifest.xml . 

My changes are similar to the existing code in category activity .

Comment (PrajwalM2212) [reply app/src/main/AndroidManifest.xml]: This pull request is solved .

**Comment by policorn** [comment]
Hi @PrajwalM2212, please fix the conflicting files

**Comment by teolemon** [comment]
Merging, thanks @PrajwalM2212 :)

**Pull request merged**